resources/cmake/ConnextDdsCodegen.cmake

```diff
@@ -633,6 +634,10 @@ function(connextdds_rtiddsgen_run)
         list(APPEND extra_flags "-example" ${CONNEXTDDS_ARCH})
     endif()
 
+    if(_CODEGEN_STRING_SIZE)
```

Comment:
Can you add the new parameter to the [documentation](https://github.com/rticommunity/rticonnextdds-examples/blob/1679cb84e67841287ee0d764e14124caa150cd6e/resources/cmake/ConnextDdsCodegen.cmake#L23-L109) similarly to the other ones?